testsuite/tests/kuadrant/gateway/conftest.py

```diff
@@ -0,0 +1,17 @@
+"""Conftest for gateway tests"""
+import pytest
+
+from testsuite.openshift.objects.auth_config.auth_policy import AuthPolicy
+
+
+@pytest.fixture(scope="module")
+def authorization(gateway, authorization_name, openshift, module_label):
+    """Create AuthPolicy attached to gateway"""
+    return AuthPolicy.create_instance(openshift, authorization_name, gateway, labels={"testRun": module_label})
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, authorization):
```

Comment:
why is this override needed?

Reply:
I don't need a second component, `RateLimitPolicy`, here

Reply:
I would override Ratelimit fixture and return None then, we do not need to limit ourselves to AuthPolicy in all gateway tests

Reply:
Can't it be done in the feature tests then? Overriding commit for this purpose looks like the conventional way in the testsuite now.

Reply:
Well, the issue is not that you are rewriting commit (even though overriding the ratelimit fixture is much much nicer), the issue is you are doing it in the conftest for all future gateway tests